core/templates/components/forms/schema-based-editors/schema-based-editor.component.html

```diff
@@ -1,13 +1,13 @@
 <form #hybridForm="ngForm">
-  <schema-based-choices-editor *ngIf="schema.choices"
+  <schema-based-choices-editor *ngIf="schema['choices']"
```

Comment:
schema type is set as unknown, this is a workaround to get the angular template ts checks to pass.